### PR TITLE
Add COS integration

### PR DESCRIPTION
**Pull request by natalian98**
This PR adds integration with Canonical Observability Stack.

### CODE REVIEW

**Comment by natalian98** [comment]
needs https://github.com/canonical/oathkeeper-rock/pull/69 to be merged first

File: src/charm.py [hunk 1]
```diff
@@ -115,8 +128,40 @@ def __init__(self, *args) -> None:
...
+                }
+            ],
+        )
+
+        self.loki_consumer = LogProxyConsumer(
+            self,
+            log_files=[str(LOG_PATH)],
+            relation_name=LOKI_PUSH_API_RELATION_NAME,
+            container_name=self._container_name,
+        )
```
Comment (nsklikas) [inline src/charm.py:151]: We should use LogForwarder (see https://github.com/canonical/openfga-operator/blob/main/src/charm.py#L111)

File: src/charm.py [hunk 2]
```diff
@@ -198,14 +248,27 @@ def _oathkeeper_layer(self) -> Layer:
...
+
         layer_config = {
             "summary": "oathkeeper-operator layer",
             "description": "pebble config layer for oathkeeper-operator",
             "services": {
                 self._service_name: {
                     "override": "replace",
                     "summary": "Oathkeeper Operator layer",
-                    "command": f"oathkeeper serve -c {self._oathkeeper_config_file_path}",
+                    "command": '/bin/sh -c "{} {} 2>&1 | tee -a {}"'.format(
```
Comment (nsklikas) [inline src/charm.py:267]: This trick is no longer needed

**Review by nsklikas — COMMENTED** [review]
One comment

**Review by nsklikas — APPROVED** [review]
I didn't check the dashboards, but lgtm

**Pull request merged**